build-system/release-tagger/update-issue-tracker.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @fileoverview
+ * Create or update release issue tracker
+ */
+
+const dedent = require('dedent');
+const {createIssue, getIssue, updateIssue} = require('./utils');
+
+const promotions = {
+  'beta-opt-in': (tag) =>
+    `- [ ] <!-- tag=${tag} channel=beta-opt-in -->${tag} promoted to Experimental and Beta (opt-in) channels (PROMOTE_TIME)`,
+  'beta-percent': (tag) =>
+    `- [ ] <!-- tag=${tag} channel=beta-percent -->${tag} promoted to Experimental and Beta (1% traffic) channels (PROMOTE_TIME)`,
+  'stable': (tag) =>
+    `- [ ] <!-- tag=${tag} channel=stable -->${tag} promoted to Stable channel (PROMOTE_TIME)`,
+  'lts': (tag) =>
+    `- [ ] <!-- tag=${tag} channel=lts -->(optional) ${tag} promoted to LTS channel (PROMOTE_TIME)`,
+};
```

Comment:
This feels a bit "magicy" and does some repetition. I think I'd rather this be split into a function and a const that only contains the changing bits - something like:
```suggestion
const CHANNEL_NAMES = {
  'beta-opt-in': 'Experimental and Beta (opt-in) channels',
  'beta-percent': 'Experimental and Beta (1% traffic) channels',
  'stable': 'Stable channel',
  // Not sure how to handle the "optional" part for now :)
  'lts': 'LTS channel',  
};

/**
 * @param {string} channel
 * @param {string} tag
 * @return {string}
 */
function generateTask(channel, tag) {
  return `- [ ] <!-- tag=${tag} channel=${channel} -->${tag} promoted to ${CHANNEL_NAMES[channel]) (PROMOTE_TIME)`;
}
```

Reply:
Good suggestion thank you!